content/blog/2020-11-praktikum-und-react/praktikum-und-react.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Erste Schritte in der Informatik'
+slug: intern-and-react
```

Comment:
Ich hätte hier auch eher "praktikum-und-react" als slug genommen. Wir haben sonst immer Deutsche Slugs

Reply:
oder sogar besser gleich wie der Titel "erste-schritte-in-der-informatik"